sdk/cmake/CMakeMacros.cmake

```diff
@@ -540,6 +544,12 @@ else()
             set_target_properties(${name} PROPERTIES DEFINE_SYMBOL "")
         endif()
     endfunction()
+
+    function(add_executable name)
```

Comment:
This does not seem worth doing just to clear out `DEFINE_SYMBOLS` -- overriding builtins has significant performance implications in CMake.

Reply:
Maybe, maybe not... anyway `add_library` is already overloaded similarly for the same purpose.